assets/js/directives/bc-file-upload.directive.js

```diff
@@ -33,15 +31,22 @@ function bcFileUpload ($rootScope, Alerts) {
 
     scope.upload = () => {
       scope.disableWebcam();
-      scope.enableWebcam();
       scope.onUpload();
     };
 
     scope.webcamError = () => {
-      scope.disableWebcam();
+      scope.state.webcam.error = true;
       Alerts.displayError('CAMERA_PERMISSION_DENIED');
     };
 
+    scope.webcam = {
+      video: {},
+      videoWidth: 660,
+      videoHeight: 495
+    };
+
+    scope.canvasOpts = { x: 0, y: 70, w: 660, h: 355 };
```

Comment:
These numbers are not arbitrary. The webcam stream overflows its container to match the design. The canvasOpts crop the actual webcam stream to match what the user sees (and 'captures').